scripts/ci/ensure-all-node-requirements-available.sh

```diff
@@ -91,7 +91,7 @@ git ls-tree -r HEAD | while IFS='' read line; do
             done
           fi
           # Check optionalDependencies from package.json
-          if [[ $requirement_found -eq 0 ]]; then
+          if [[ $requirement_found -eq 0 ]] && [[ -n "${NPM_OPTIONAL_MODULES:-}" ]]; then
```

Comment:
Yeah, I think you should send a different PR with these changes (I have no idea what I'm doing), and I'll rebase afterwards.